### ISSUE TITLE
bug: HOSTNAME_EXTERNAL does not work with SQS ASF

### ISSUE THREAD

**Issue by kandrejevs2** [localstack/localstack#5268]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

sqs getQueueUrl returns URL with localhost domain if `PROVIDER_OVERRIDE_SQS=asf` is used. 

### Expected Behavior

HOSTNAME_EXTERNAL should override localhost domain for asf queues

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    with docker-compose

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

with aws php SDK:
```php
try {
    $result = $client->listQueues();
    foreach ($result->get('QueueUrls') as $queueUrl) {
        echo "$queueUrl\n";
    }
} catch (AwsException $e) {
    // output error message if fails
    error_log($e->getMessage());
}
```

### Environment

```markdown
- OS: docker-compose on ubuntu 20.04
- LocalStack: 0.13.3
```

### Anything else?

when disabling asf override it works as expected.

**Comment by dfangl** [comment]
Hi @kandrejevs2 !
This should have been fixed with #5238 ! Can you pull the latest image and see if this fixes it for you? Thanks!

**Comment by kandrejevs2** [comment]
thanks @dfangl . latest seems to be working as expected. weirdly I thought 13.3 is the latest release.
